ios/keyman/Keyman/Keyman.xcodeproj/project.pbxproj

```diff
@@ -957,6 +959,41 @@
 			shellPath = /bin/sh;
 			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint --config ../../.swiftlint.yml\nelse\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
 		};
+		CE7E13D8240E285800252C00 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = ". $KEYMAN_ROOT/resources/build/xcode-utils.sh\n\n# Calls resource script to update build products' versioning.\n# true:  applies VERSION_WITH_TAG to custom KeymanVersionWithTag plist member used for in-app display\nphaseSetBundleVersions true\n\n# Also updates the version string for Settings\nsetSettingsBundleVersion\n";
```

Comment:
I see `phaseSetBundleVersions()` takes a parameter `true`, but I don't understand what the first `true` corresponds to.

Reply:
Please see the (code) comment at https://github.com/keymanapp/keyman/pull/2775/files#diff-f5d9f24e4c3e3aaaa2103de4ea7d4818R6

Reply:
Don't forget to quote `$KEYMAN_ROOT` and any other variables here...

Reply:
quotes needed